tests/backend/test_multimode_state_preparations.py

```diff
@@ -137,7 +137,7 @@ def test_compare_single_mode_and_multimode_dm_preparation(
         multi_mode_preparation_31_probs = np.array(backend.state(modes=[3, 1]).all_fock_probs())
 
         single_mode_vac = np.zeros((cutoff, cutoff), dtype=np.complex128)
-        single_mode_vac.itemset(0, 1.0 + 0.0j)
+        single_mode_vac[0, 0] = 1.0 + 0.0j
```

Comment:
Why `[0, 0]` and not `[0]`?

Reply:
because it is a 2D array (see line 139), and we want to set the first element (upper left corner). This is at least what `itemset` did. I first had `[0]` here and the test failed.